cpu/esp32/esp-idf/esp_idf_support.c

```diff
@@ -111,7 +111,7 @@ void IRAM_ATTR esp_log_writev(esp_log_level_t level,
      * We use the log level set for the given tag instead of using
      * the given log level.
      */
-    esp_log_level_t act_level = LOG_DEBUG;
+    esp_log_level_t act_level = (esp_log_level_t)LOG_DEBUG;
```

Comment:
@maribu I'm wondering what the cleaner approach would be here. Casting to RIOT's `LOG_DEBUG` to `esp_log_level_t` or using `ESP_LOG_DEBUG` as in PR #18532. the enumeration `esp_log_level_t` is compatible with RIOT's log levels.

Reply:
I think it is more a matter of personal taste an either is fine